### ISSUE TITLE
how do we hide url and show only the web content?

### ISSUE THREAD

**Issue by T3Z3N** [EddyVerbruggen/cordova-plugin-safariviewcontroller#158]
how do we hide url and show only the web content?

**Comment by allnash** [comment]
Can you please apply the following?
``
let safariViewController = SFSafariViewController(URL: url)
presentViewController(safariViewController, animated: true) {
    var frame = safariViewController.view.frame
    let OffsetY: CGFloat  = 64
    frame.origin = CGPoint(x: frame.origin.x, y: frame.origin.y - OffsetY)
    frame.size = CGSize(width: frame.width, height: frame.height + OffsetY)
    safariViewController.view.frame = frame
}
```

```
presentViewController(svc, animated: true) {
    let width: CGFloat = 66
    let x: CGFloat = self.view.frame.width - width

    // It can be any overlay. May be your logo image here inside an imageView.
    let overlay = UIView(frame: CGRect(x: x, y: 20, width: width, height: 44))
    overlay.backgroundColor = UIColor.blackColor().colorWithAlphaComponent(0.5)
    svc.view.addSubview(overlay)
}
```

**Comment by allnash** [comment]
@EddyVerbruggen

**Comment by EddyVerbruggen** [comment]
I'd be happy to merge a PR! This looks like Swift, so I can't simply paste that into the project. And you may want to make it configurable through the JS API..

**Comment by allnash** [comment]
True. I am a little rusty on my ObjC. I can attempt it in the next month or so.

**Comment by Scobee** [comment]
@EddyVerbruggen was this ever implemented ?

**Comment by EddyVerbruggen** [comment]
No
